src/lib.rs

```diff
@@ -183,7 +187,7 @@ impl AsyncDb {
             .append_pair("q", query);
 
         let response =
-            self.client.get(query_endpoint)
+            self.client.get(hyper::Uri::from_str(query_endpoint.as_str()).unwrap())
```

Comment:
I'd go ahead and change both `unwrap`s to `expect`s with a somewhat understandable error message.